### DISCUSSION TITLE
Quasar 2 migration documentation

### DISCUSSION THREAD

**Discussion by pytely** [quasarframework/quasar#10608] [Ideas / Proposals] [answered]
The is the code for the guide
`export default ({ app }) => {
  // Set i18n instance on app
  app.use(i18n)
}`
should it be?
`export default boot({ app }) => {
  // Set i18n instance on app
  app.use(i18n)
})`

**Comment by hawkeye64** [comment, marked as answer]
Yes, especially for typescript users

**Reply by rstoenescu** [reply]
The `boot` function is just a wrapper for TS users (and IDE autocompletion, eg VSCode). It's essentially a no-op wrapper.
